crates/fuel-gas-price-algorithm/src/v1.rs

```diff
@@ -540,6 +548,12 @@ impl AlgorithmUpdaterV1 {
             .saturating_mul(self.gas_price_factor.into())
     }
 
+    fn max_scaled_da_gas_price(&self) -> u64 {
+        // note: here we make sure that a correct maximum is used; hacky :(
+        max(self.max_da_gas_price, self.min_da_gas_price + 1)
```

Comment:
Probably can just use `min_da_gas_price` without the `+ 1`, but maybe there is a weird edge case I missing.

Reply:
yup, detected and fixed

Reply:
addressed in 4c42380da7